### PR TITLE
V3 actual consistent naming

### PR DESCRIPTION
**Pull request by bzz**
Refactoring for clarity and consistency, as discussed IRL:
- [x] NodeExtType => PyNodeExtType, NodeExt => PyNodeExt
- [x] PyUastType => PyContextType, PyUast => PyContext

Part of the https://github.com/bblfsh/client-python/pull/128

### CODE REVIEW

File: bblfsh/pyuast.cc [hunk 1]
```diff
@@ -954,17 +954,17 @@ static PyObject *PyUastIter_new(PyObject *self, PyObject *args) {
...
+    auto node = (PyNodeExt*)obj;
     return node->ctx->Iterate(obj, (TreeOrder)order);
   }
   // Python object -> create a new context and attach it to an iterator
   Context* ctx = new Context();
   return ctx->Iterate(obj, (TreeOrder)order, true);
 }
 
-static PyObject *PyUastDecode(PyObject *self, PyObject *args) {
+static PyObject *PyContextDecode(PyObject *self, PyObject *args) {
```
Comment (dennwc) [inline bblfsh/pyuast.cc]: `Py_Decode`, since it's a global method, or `PyContextExt_decode` since it's a constructor specifically for `PyContextExt`

Comment (bzz) [inline bblfsh/pyuast.cc]:
```suggestion
static PyObject *PyContextExt_decode(PyObject *self, PyObject *args) {
```

Comment (bzz) [reply bblfsh/pyuast.cc]: 👍 this is a [perfect example](https://github.com/dennwc/client-python/pull/1/files#r225858481) how we could benefit from new github feature :) https://blog.github.com/category/all/#suggested-changes-public-beta

Comment (bzz) [reply bblfsh/pyuast.cc]: addressed in 2785a60

**Review by dennwc — APPROVED** [review]
Looks good, thanks! :)

**Comment by bzz** [comment]
🎉  :shipit:

**Pull request merged**